chia/server/ws_connection.py

```diff
@@ -53,6 +54,19 @@ def __init__(
         # Local properties
         self.ws: Any = ws
         self.local_type = local_type
+        self.annotations: Dict[str, Type[Streamable]] = {}
```

Comment:
```suggestion
        self.request_types: Dict[str, Type[Streamable]] = {}
```

Maybe? Or something else but `annotations` isn't really meaningful here imo.

Reply:
Agreed and fixed per your suggestion.